backend/ctgov/api/views.py

```diff
@@ -2,12 +2,96 @@
 from rest_framework.response import Response
 from rest_framework import status
 from rest_framework import permissions
-from ctgov.models import BriefSummaries
-from .serializers import BriefSummariesSerializer
+from ctgov.models import BriefSummaries, BasicSearch, Studies, Facilities
+from .serializers import (
+    BriefSummariesSerializer,
+    BasicSearchSerializer,
+    CountriesSerializer,
+)
+from django.db.models import Q
 
 
 class BriefSummariesListApiView(APIView):
     def get(self, request, *args, **kwargs):
         summaries = BriefSummaries.objects.all()[:10]
         serializer = BriefSummariesSerializer(summaries, many=True)
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class CountriesListApiView(APIView):
+    def get(self, request, *args, **kwargs):
+        countries = (
+            Facilities.objects.all()
```

Comment:
Facilities contains 2,454,268 records. Not sure how the ORM optimizes the query based on the distinct, but this seems a bit much to query just to return 200 rows of data. I know this may get resolved without getting addressed (due to interest of time), but I feel like I have to bring it up.

Reply:
It was in fact very quick and you can run the api to check. I think it is in milli-seconds. But, we can always look for better options.